ui/app/AppLayouts/Chat/ContactsColumn.qml

```diff
@@ -291,21 +291,70 @@ Item {
                         }
 
                         Rectangle {
-                            id: contactImage
+                          id: contactImage
+                          width: 40
+                          height: 40
+                          anchors.left: parent.left
+                          anchors.leftMargin: Theme.padding
+                          anchors.top: parent.top
+                          anchors.topMargin: 12
+                          anchors.bottom: parent.bottom
+                          anchors.bottomMargin: 12
+                          radius: 50
+
+                          Loader {
+                            sourceComponent: chatType == 1 ? imageIdenticon : letterIdenticon
+                            anchors.fill: parent
+                          }
+
+                          Component {
+                            id: letterIdenticon
+                            Rectangle {
+                              width: 40
+                              height: 40
+                              radius: 50
+                              color: {
+                                  const color = chatsModel.getChannelColor(name)
+                                  if (!color) {
+                                      return Theme.transparent
+                                  }
+                                  return color
+                              }
+
+                              Text {
+                                text: (name.charAt(0) == "#" ? name.charAt(1) : name.charAt(0)).toUpperCase()
+                                opacity: 0.7
+                                font.weight: Font.Bold
+                                font.pixelSize: 21
+                                color: "white"
+                                anchors.horizontalCenter: parent.horizontalCenter
+                                anchors.verticalCenter: parent.verticalCenter
+                              }
+                            }
+                          }
+                        }
```

Comment:
Since we use this twice, would be nice to move it to a reusable component in `shared/`